library/std/src/primitive_docs.rs

```diff
@@ -98,7 +98,7 @@ mod prim_bool {}
 /// at all we know it can never produce a value which isn't a [`u32`]. This illustrates another
 /// behaviour of the `!` type - expressions with type `!` will coerce into any other type.
```

Comment:
```suggestion
/// behavior of the `!` type - expressions with type `!` will coerce into any other type.
```
it seems to be the case that Rust is en_US standardized, though I can't provide a reference to the rules for that

Reply:
Please make this change in a separate PR, it doesn't belong in this one.

Reply:
Different parts of Rust use different spelling, so I would not say that Rust uses a standardized spelling.

Reply:
LGTM tho something like the `Rust Reference` uses only en_US as an example